code/modules/jobs/job_types/roboticist.dm

```diff
@@ -19,6 +19,12 @@
 
 	display_order = JOB_DISPLAY_ORDER_ROBOTICIST
 
+	mail_goodies = list(
+		/obj/item/assembly/flash/handheld = 30,
```

Comment:
robos can just print flashes themselves for extremely cheap and in bulk, this might be one of the worst mail items to recieve just because of how useless it is to get

Reply:
replace it with, like, a posibrain or something

or a really fancy piece of mech equipment

or some of the caravan tools

or diagnostic sunglasses

or all of the above